source/api_reference/search_payments_reference/index.html.md.erb

```diff
@@ -110,7 +110,7 @@ After you send your request to search payments, you’ll receive a response simi
           "country": "GB"
         },
         "card_brand": "Visa",
-        "card_type": "credit"
+        "card_type": "credit",
```

Comment:
This looks like a mistake.

Reply:
Whoops, good spot. I'd initially added the `wallet_type` attribute here but then took it out because it didn't work with the example. Forgot to remove the comma.